src/app/modules/color-scheme/color-scheme-contents/color-scheme-contents.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="color-scheme-contents-container">
+  <div id="scheme-contents" class="scheme-contents">
+    <ccf-color-bar #colorbar *ngFor="let i of [0,1,2,3,4,5,6,7]" class="swatch" 
```

Comment:
Iterate over the color schemes (check color bar comments for how to access index in for loop)